replicated-security/neutron-rehearsal-1/README.md

```diff
@@ -0,0 +1,113 @@
+
+* **Reference binary**: [neutrond-linux-amd64](./neutrond-linux-amd64)
+* **Binary sha256sum**: `394736d71f5798d0de3427aac96fba27f8df07c23d8d47eedd837774f4e4cb48`
+* **Genesis file _without CCV state_:** [neutron-rehearsal-1-genesis-without-ccv.json](neutron-rehearsal-1-genesis-without-ccv.json), verify with `shasum -a 256 neutron-rehearsal-1-genesis-without-ccv.json`
+* **SHA256 for genesis file _without CCV state_**: `e5a5e9f8ba69a846d18737c2d2a6a6de0250f45c53c07df274dc80ab9bda2ba0`
+
+
+* Genesis file hash
+  * The SHA256 is used to verify against the genesis file (without CCV state) that the proposer has made available for review.
+  * The `neutron-rehearsal-1-genesis-without-ccv.json` file cannot be used to run the chain: it must be updated with the CCV (Cross Chain Validation) state after the spawn time is reached.
+  * The genesis file includes funds for a relayer and a faucet account as well as account with funds for different internal needs, `signed_blocks_window` has been set to `864000`, and `min_signed_per_window` has been set to `5%`.
+* Binary hash
+  * The `neutrond-linux-amd64` binary is only provided to verify the SHA256. It was built with Interchain Security release [`v0.4.0`](https://github.com/neutron-org/neutron/releases/tag/v0.4.0). You can generate the binary following the build instructions in that repo or using one of the scripts provided here.
+* Spawn time
+  * Even if a proposal passes, the CCV state will not be available from the provider chain until after the spawn time is reached.
+
+> If you want to build genesis by your own please take into account difference between `ccvconsumer` section generated by current version of `gaiad` and `ccvconsumer` required for neutron to run. Please check and add `"soft_opt_out_threshold": "0.05"` to the `params` section if it absent.
+
+For more information regarding the consumer chain creation process, see [CCV: Overview and Basic Concepts](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md).
+
+## Endpoints
+
+Endpoints are exposed as subdomains for the sentry and snapshot nodes (described below) as follows:
+
+* `https://rpc.rehearsal.ntrn.tech`
+* `https://grpc.rehearsal.ntrn.tech`
+* `https://rest.rehearsal.ntrn.tech`
+
+Seed nodes:
+
+1. `e2c07e8e6e808fb36cca0fc580e31216772841df@p2p.rehearsal.ntrn.tech:26656`
+
+The following state sync node serve snapshots every 1000 blocks:
+
+1. `https://rpc.rehearsal.ntrn.tech:443`
+
+## IBC Information
+
+Connections and channels will be posted here shortly after the chain launches.
+
+### Clients
+
+* `07-tendermint-0`
+  * Counterparty: [`provider`](/replicated-security/provider/README.md) `07-tendermint-1`
+
+### Channels
+
+* `channel-0`: consumer port
+  * Counterparty: [`provider`](/replicated-security/provider/README.md) `channel-2`
+* `channel-1`: transfer port
+  * Counterparty: [`provider`](/replicated-security/provider/README.md) `channel-3`
```

Comment:
You can remove these lines, I'll update this section after the chain becomes interchain-secured.